src/utils/postcomictodiscord.js

```diff
@@ -0,0 +1,26 @@
+async function PostComicToDiscord(client, channelId, comicDetail) {
+  // await client.channels.cache.get(channelId).send({
+  //   content: `Title: ${comicDetail.Title}\n Number: ${comicDetail.Num}\n Link: <${comicDetail.Url}>`,
+  //   files: [{
+  //     attachment: String(comicDetail.Img),
+  //     name: 'file.jpg',
+  //     description: String(comicDetail.Alt_text),
+  //   }],
+  // });
```

Comment:
consider deleting commented code. it's a little messy to leave stuff in there, unless you need it for some reason
```suggestion
```